src/document.rs

```diff
@@ -184,18 +185,28 @@ impl Display for DocumentClass {
             DocumentClass::Book => write!(f, "book"),
             DocumentClass::Report => write!(f, "report"),
             DocumentClass::Part => write!(f, ""),
+            DocumentClass::Other(ref s) => write!(f, "{}", *s),
         }
     }
 }
 
+/// An element of the document's preamble.
+#[derive(Clone, Debug, PartialEq)]
+pub enum PreambleElement {
+  /// Use a package with an optional argument.
+  UsePackage(Option<String>, String),
```

Comment:
Would it be more readable if we define this as `UsePackage { package: String, argument: Option<String> }`?

Reply:
That seems fine, I also added allow(missing_docs) to avoid having to document both package: and argument: and UsePackage, which otherwise would force a rather simple thing to be fairly verbose.